learn/advanced/dumps.md

```diff
@@ -52,6 +54,10 @@ If a dump file is visible in the file system, the dump process was successfully
 
 When a dump is being imported, the API is not available to the task queue. As a result, no read or write operations can be performed until the importing process is complete.
 
+During a dump import, all indexes contained in the indicated `.dump` file are imported along with their associated documents and settings. Any existing index with the same `uid` as an index in the dump file will be overwritten.
+
+Dump imports must be performed when launching a Meilisearch instance [using the `import-dump` command-line option](/learn/configuration/instance_options.md#import-dump).
```

Comment:
Could you make this the first line in this section, please? In a section called "Importing a dump" we gotta start by explaining how to import a dump before providing further details.